lib/elixir/lib/inspect/algebra.ex

```diff
@@ -175,6 +185,11 @@ defmodule Inspect.Algebra do
     quote do: {:doc_group, unquote(group)}
   end
 
+  @typep doc_color :: {:doc_color, t, IO.ANSI.ansidata}
+  defmacrop doc_color(doc, color) do
+    quote do: {:doc_color, unquote(doc), unquote(color)}
+  end
```

Comment:
@ericmj we use them for pattern matching too and inline functions won't work for that.